Papers by Eva Hajičová: diacritics, orcid, other small changes (pages, abstract, title)

weissenh opened the pull request:
Papers by Eva Hajičová: added diacritics to name incl. co-authors, added orcid, small other changes

- went through [EHs papers](https://aclanthology.org/people/eva-hajicova/) and added diacritics to her name ONLY IF they also appear in the paper PDF (and for co-authors too)
- added missing abstracts and page numbers for some of those papers while I am at it
- changed three of the paper titles to conform with title in the PDF
  - [J95-1004](https://aclanthology.org/J95-1004/) (title maybe confused with 1993 paper?) 
  - [LREC14 paper](https://aclanthology.org/L14-1337/) (superfluous quotation mark at end)
  - [W07-1530](https://aclanthology.org/W07-1530/) on PDF it starts with "Panel Discussion", in Proceedings preface it is the old title starting with "Discourse Annotation"
- added `orcid` attribute to EH's `author` tag ONLY IF the paper is also listed on [her ORCID page](https://orcid.org/0000-0002-8640-246X) (not all of them are currently)

I've decided against using the 'fix data' button (which would have created multiple separate GitHub issues), although I guess it is easier to verify with the bot posting a picture of the first page? Let me know if there is anything I can do which would make reviewing my changes easier.

I've tried to follow [wiki/Issuing-Pull-Requests-For-Corrections](wiki/Issuing-Pull-Requests-For-Corrections). Not sure whether I need to change any other files?  haven't checked what happens to co-author pages if they have just this one or multiple papers - is there anything I need to check regarding name variants? do I need to verify I haven't mistakenly introduced any special characters that break rendering?

I'm not associated with EH in any way, it just bugged me that many times the diacritics were missing and oh my that was kind of a rabbit hole :)

> (Please replace this text with a description of the changes effected by this pull request.
Include a link to the corresponding Github Issue, if there is one.
Details on how to do this ([can be found here](https://help.github.com/articles/basic-writing-and-formatting-syntax/#referencing-issues-and-pull-requests)).)

Haven't opened an issue for this, so nothing to link. Description of changes see above.

nschneid commented:
> we wanted the semantics of "ORCID at the paper-level" to be that it was the submission materials (e.g. from OpenReview) that contained this ORCID. Can @mjpost @nschneid weigh in if I remember this right?

My thinking was that the ORCID at the paper level should only be added manually, not heuristically. Currently this means it was present in the ingested data (specified manually by the author in their OpenID or SoftConf account). Maybe it would be best to leave it that way for now, and move the ORCID changes into a separate PR that can be merged later.

mjpost commented:
I second @mbollmann's gratitude—thanks for the extensive work here! I love to see these correction efforts and am grateful for it.

I don't know that it's written down, but my understanding is the same as @nschneid's, that we do plan to reserve paper-level `orcid` attributes as having been present at ingestion. Since we're in this lingering transition period, I'm flexible enough that we could abuse it this one time, but I suppose the proper way to do this is to record the ORCID iD in the name variants file (which you should add anyway), and then replace these `orcid` attributes with an `id` attribute pointing to the author's entry. Marcel can correct me, but I think that will preserve the semantics of these tags while enabling a smooth transition to the new representation (which has been held up, mostly because of me...)

weissenh commented:
I've pushed the changes now (option b):
- I added her [orcid](https://orcid.org/0000-0002-8640-246X) in the `name_variants.yaml`
- I deleted all paper-level orcid-mentions I introduced earlier. I did not introduce any `id` attributes (I assume there is only one Eva Hajičová).

I've also checked the papers (around 50) and based on title (research topic), affiliation and co-authors I'm pretty certain they all belong to the said person and no other person with the same name. Thankfully her affiliation (Charles University, Prague) never changed so was easy to check.

From my point of view, the PR is now ready to be reviewed and merged.
Let me know if you have any further questions or if there is anything more I can do to make reviewing easier for you.

mjpost commented:
I just approved and unblocked the build check. We should be good to go if that comes through.
